SpatialGDK/Source/SpatialGDK/Public/Interop/ActorSetWriter.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Schema/ActorSetMember.h"
+#include "SpatialCommonTypes.h"
+#include "SpatialView/EntityComponentTypes.h"
+
+class USpatialPackageMapClient;
+
+namespace SpatialGDK
+{
+class ViewCoordinator;
+
+class ActorSetWriter
```

Comment:
Happy to discuss this offline but I think this is better served as two functions than a stack only class.
The first function only requires the package map and the second one only requires the coordinator. However the second one might make for slightly more explict code if it just returns the update and so doesn't need the coordinator either.

So it could be one function that takes an AActor and a PackageMap, and one that takes an entity ID and an AActor.